RocketChat/Rocket.Chat
# [NEW] Warning Popup reinforcing Zapier deprecation on Integrations Admin settings

## Proposed changes (including videos or screenshots)


#### Warning being displayed on Zapier settings, above Zapier itself:
![warning](https://user-images.githubusercontent.com/47800334/213696834-be7ac8e0-7249-4d02-93ba-e4c5e34b67da.png)





## Issue(s)


## Steps to test or reproduce

On an Admin Workspace:

1. Go to workspace administration settings
2. Click on "Integrations" in the sidebar
3. Go to the "Zapier" tab
4. The new Popup should show

## Further comments

A new message (in English) was written on the translation JSON used by the `useTranslation()` hook, and it still needs to be written on other languages as well. Note that the proposed warning popup it's following Fuselage's current design pattern, so it may differ from the initial request and from the <a href="https://rocketchat.github.io/fuselage/fuselage/master/?path=/docs/feedback-banner--normal">storybook design SHOWN HERE</a>.
